collective/searchandreplace/interfaces.py

```diff
@@ -65,3 +65,12 @@ class ISearchReplaceSettings(Interface):
         required=False,
         default=50,
     )
+
+    update_modified = schema.Bool(
+        title=_(u'Update the modified datetime when replacing.'),
+        description=_(
+            u'If checked, the modified index/metadata of the object'
+            u' having text replaced will be updated.'),
```

Comment:
Minor issue: instead of a leading space on this line, please add a trailing space in the string in the previous line. That is what the description in the field above it does (and is also what I am used to).

(Remark: the description of the maximum_text_characters field does not have a unicode marker at the beginning, but I prefer having such a marker in all lines, just like you do here.)